FireboltDotNetSdk.Tests/Integration/ConnectionTest.cs

```diff
@@ -89,6 +91,7 @@ public void InvalidAccountConnectTest(string errorMessage)
         }
 
         [Test]
+        [Category("general")]
```

Comment:
I agree. The problem is that there are 4 categories of integration tests now. 
1. plain integration tests marked with category "Integration" at the class level. 
2. v1
3. v2
4. engine-v2
When I run test for each explicit category I have to skip tests that belong to others. For example 
```
Catetory=Integration&Category=v1&Category!=v2&Category!=engine-v2
```

This will run all integration tests that belong to category v1 and ignore v2 and egnne-v2. The problem is that there are a lot of tests that should run for 2 categories, e.g. v1 and v2. These tests will not be executed. 

So, this "general" category was the simplest (but dangerous) solution that I have found. I am very glad that you payed attention on this and will appreciate if you have better ideas how to solve this.

Reply:
Can those "general" tests be refactored into their own class? At least this way we'll be able to set this category once per class than per test.

Reply:
Well, the general tests are distributed among 3 test cases. I can split each test case into 2: General and EnvironmentSpecific, e.g.
1. ExecutionTest -> GeneralExecutionTest & EnvironementSpecificExecutionTest
2. SystemEngineTest -> GeneralSystemEngineTest & EnvironementSpecificSystemEngineTest
3. ConnectionTest -> GeneralConnectionTest & EnvironementSpecificConnectionTest

But IMHO this will be more annoying. For example think about test that was general and not it works differently on v1 and v2. I will have to move this test to another class instead of just marking it with different annotation. 

I can suggest something else. I can validate that there are not uncategorized integration tests. I can do this somewhere into the `IntegrationTest` base class. I know how to do something similar in java (where we do not need this now) but do not know how to do this in C# on the top of my head, but I am sure it is possible. If you agree that this is preferable, I'd suggest to do this in separate commit.

Reply:
created separate issue https://packboard.atlassian.net/browse/FIR-32172